actionlist/app/reducers/action_items.js

```diff
@@ -0,0 +1,29 @@
+/* eslint import/no-extraneous-dependencies: ["error", {"peerDependencies": true}] */
```

Comment:
This directive shouldn't be needed anymore (see my other comment on a similar line)